api/apps/geoprocessing/test/integration/marxan-run/marxan-run.e2e-spec.ts

```diff
@@ -52,17 +81,35 @@ afterAll(async () => {
   });
 });
 
-// TODO: later it may be disposed and use "real" implementation
-// for now, as we don't know yet the values/inputs, we simply
-// copy sample input provided by Marxan
-@Injectable()
-class SampleMarxanInputFilesCreator implements InputFiles {
-  include(values: unknown, directory: string): Promise<void> {
-    copySync(
-      process.cwd() +
-        `/apps/geoprocessing/src/marxan-sandboxed-runner/__mocks__/sample-input`,
-      directory,
-    );
-    return Promise.resolve(undefined);
-  }
-}
+const resources = [
+  {
+    name: `input.dat`,
+    assetUrl: `http://localhost:3000/input.dat`,
+    targetRelativeDestination: `input.dat`,
+  },
+  {
+    name: `pu.dat`,
+    assetUrl: `http://localhost:3000/pu.dat`,
+    targetRelativeDestination: `input/pu.dat`,
+  },
+  {
+    name: `spec.dat`,
+    assetUrl: `http://localhost:3000/spec.dat`,
+    targetRelativeDestination: `input/spec.dat`,
+  },
+  {
+    name: `puvsp.dat`,
+    assetUrl: `http://localhost:3000/puvsp.dat`,
+    targetRelativeDestination: `input/puvsp.dat`,
+  },
+  {
+    name: `puvsp_sporder.dat`,
+    assetUrl: `http://localhost:3000/puvsp_sporder.dat`,
+    targetRelativeDestination: `input/puvsp_sporder.dat`,
```

Comment:
`puvspr*` (missing the final `r`)

Reply:
@hotzevzl 
![image](https://user-images.githubusercontent.com/3466388/123798992-08685680-d8e8-11eb-87b0-0faebc9279c9.png)

could be but it was taken directly from marxan's example ; )

Reply:
@kgajowy which warez site did you download this copy of Marxan from? 😛 
they're listed as `puvspr.dat` and `puvspr_sporder.dat` in the manual - no big deal of course as we can name these files as we wish, but just noting this to keep to the defaults mentioned in the manual

Reply:
@hotzevzl https://github.com/Marxan-source-code/marxan#test-data :)

![image](https://user-images.githubusercontent.com/3466388/123915387-bd4e5200-d980-11eb-82f8-a412c75e08b6.png)

🤷